convert.go

```diff
@@ -0,0 +1,140 @@
+package godash
+
+import (
+	"strconv"
+)
+
+type AnyType interface {
+	convert()
+}
+
+// *******************************************************
+// ************** Convert string to integer **************
+// *******************************************************
+
+func ToInt(str string) (int, error) {
+	// Covert a string to a base-10 integer
+	val, err := strconv.ParseInt(str, 0, 0)
```

Comment:
Might be clearer to use a `10` as the base.

Reply:
I was using `0` because it forces `ParseInt()` to interpret the base. So if we pass `"0x3A"`, it would parse the hex string to `58`. Should we however, just restrict it to integer string to `int` conversion?

Reply:
Gotcha, good call.

Reply:
I should probably add comments to explain that random `0` for the base there though